### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by pasha1932**
.

### CODE REVIEW

**Comment by pasha1932** [comment]
- [DEMO LINK](https://pasha1932.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://pasha1932.github.io/layout_product-cards/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,39 @@
...
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="card"
+      class="card__image"
+      >
```
Comment (sTorba24) [inline src/index.html:19]:
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="card"
        class="card__image"
      >
```

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $second-accent;
+    font-family: Roboto, sans-serif;
```
Comment (sTorba24) [inline src/styles/blocks/card.scss:29]: set it for body or html

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```
Comment (sTorba24) [inline src/styles/blocks/card.scss:32]: it's default values, can be removed

File: src/styles/blocks/card.scss (hunk 2 above)
Comment (sTorba24) [reply src/styles/blocks/card.scss:29]: and remove then(fix everywhere(

File: src/styles/blocks/stars.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (sTorba24) [inline src/styles/blocks/stars.scss:8]: background-shorthand can be used

File: src/styles/blocks/stars.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```
Comment (sTorba24) [inline src/styles/blocks/stars.scss:16]: implement all modifiers from task `stars`

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! Lets improve it

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,39 @@
...
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-num">$2,199</p>
+      </div>
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >Buy</a>
```
Comment (etojeDenys) [inline src/index.html:40]: Pasted Graphic

File: src/styles/blocks/stars.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
```
Comment (etojeDenys) [inline src/styles/blocks/stars.scss:20]:
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url(../images/star-active.svg);
}
```
combine it with a comma

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**